src/components/main/mainsection-blocks/mediaTable.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { FaTable, FaTimes, FaMinusCircle } from 'react-icons/fa';
+import gravityPsd from '../../../images/main-section/IMG1.jpg';
+import psdFoil from '../../../images/main-section/IMG2.jpg';
+import woodLogo from '../../../images/main-section/IMG3.jpg';
+import UniHeader from './UniHeader';
+
+const MediaTable = () => {
+    const row = [
+        {
+            postDescImg: <img className="media-img" src={gravityPsd} alt="gravityPsd" />,
+            postDescTitle: 'Gravity PSD B-Cards',
+            postDescText: 'The classic apeach...',
+            dateYear: '26 Feb,2014',
+            dateTime: '15:20',
+        },
+        {
+            postDescImg: <img className="media-img" src={psdFoil} alt="psdFoil" />,
+            postDescTitle: 'PSD Foll Sticker',
+            postDescText: 'Lorem ipsum dolor sit amet....',
+            dateYear: '20 Feb,2014',
+            dateTime: '11:03',
+        },
+        {
+            postDescImg: <img className="media-img" src={woodLogo} alt="woodLogo" />,
+            postDescTitle: 'Wood Burning Logo',
+            postDescText: 'A fresh looking wood...',
+            dateYear: '26 Feb,2014',
+            dateTime: '15:20',
+        },
+    ];
+
+    const titl = [
```

Comment:
why do you use 2 arrays here?
need to be remade for 1 array which contains all the data necessary to display rows info